src/domain/occurrence/CreateOccurrencePage.tsx

```diff
@@ -261,27 +265,38 @@ const useInitialFormValues = (isFirstOccurrence: boolean) => {
     const initialStartsAt = searcParams.get('startsAt');
     const initialEndsAt = searcParams.get('endsAt');
 
+    const initialValues = Object.assign(
+      { ...defaultInitialValues },
+      {
+        enrolmentStart: eventData?.event?.pEvent?.enrolmentStart
+          ? new Date(eventData?.event?.pEvent?.enrolmentStart)
+          : null,
+        enrolmentEndDays: eventData?.event?.pEvent?.enrolmentEndDays || 0,
+      }
```

Comment:
Okay, I need to check that. With that solution I could manage without so many changes. I could make an alternative PR or something, let's see.

Reply:
After rethinking this, what would be the purpose of `useInitialFormValues`, if it isn't to give all the values needed by the form. I understood that on-the-fly created validation scheme that you suggested @Sartonon  would have been a solution for not to include eventData in useInitialFormValues? And that is the thing that also I requested and first thought that would be a good thing, but now I've changed my mind, because if the validation schema is created on the fly from multiple pieces, it would only make it hard to follow, what really is validated. Now everything have their place.

Or did you mean something else with this suggestion @Sartonon?

Reply:
Its purpose would be to provide initialValues from the event form for the first occurrence?

Reply:
Current solution just feels a little bit hacky, we are providing some initialValues to form that are only used for validation. It seems like enrolmentStart and enrolmentEndDays are included in the form. I remember being very confused in some other form where I saw some initial values but couldn't find the in the form and it is not very easy to see that they are only used in validation.

Reply:
Someone could easily look at the forms initialValues and just delete some of them because he/she would think that maybe those were used previously in the form but had been removed but doesnt realise that they are used in the validationSchema

Reply:
Those 2 event fields are no longer part of occurrence form. I updated the validationSchema to be create on the fly before Formik, but I'm not sure if it is what you meant.

Reply:
I think you did exactly as I described :)